docs/content/getting-started/install.md

```diff
@@ -1,15 +1,15 @@
 ---
 title: "Installation"
-description: "Download and install the hof cli tool"
-brief: "Download and install the hof cli tool"
+description: "Download and install the hof CLI tool."
+brief: "Download and install the hof CLI tool."
```

Comment:
add keywords section here

you should get a message on slack tomorrow, I tried scheduling so as not to disturb your off hours

Reply:
updated slack settings 😂 , will check on the keywords! 🔨

Reply:
added keywords 🔨 let me know your thoughts!